src/python_testing/TC_OpstateCommon.py

```diff
@@ -263,7 +267,7 @@ async def TEST_TC_OPSTATE_BASE_1_1(self, endpoint=1, cluster_revision=1, feature
                 events.OperationalError.event_id,
             ]
 
-            if self.check_pics(f"{self.test_info.pics_code}.S.E01"):
+            if self.pics_guard(self.check_pics(f"{self.test_info.pics_code}.S.E01")):
```

Comment:
This one will end up going away with a quick test plan change - event list has been officially removed from the spec, so this step will never get run. 

I opened a (very short) issue - this step can be deleted from the test plan and script. I'd say do it in a separate PR to keep the changes focused. Can do it after submitting this one.

https://github.com/project-chip/matter-test-scripts/issues/371

Reply:
Feel free to resolve this once you've read it, we can continue on the issue